ingestion/functions/parsing/brazil_paraiba/paraiba.py

```diff
@@ -0,0 +1,193 @@
+                "end": only_age
+            }
+
+def convert_confirmation_method(raw_test: str):
+    if "Clínico" or "Clinico" in raw_test: #written both ways in dataset
+        return "Clinical diagnosis"
+    elif "C. Epid" in raw_test:
+        return "Other"
+    elif "Laboratorial" in raw_test:
+        return "Other"
+    elif "swab" in raw_test.lower():
+        #swabs are reported both as Swab and SWAB
+        return "PCR test"
+    elif "teste" in raw_test.lower():
+        #this is reported as both Teste Rapido and teste rapido
+        return "Serological test"  
+    else:
+        print(f'unknown confirmation method: {raw_test}')
+        return "Unknown"
+
+def convert_preexisting_conditions(raw_commorbidities: str):
+    preexistingConditions = {}
+    if raw_commorbidities != "Sem comorbidades" or "Outros" or "Tabagismo" or "Imunossupressão":
+    	#How do we want to deal with 'Others' - do we want to have hasPreexistingConditions = True
+    	#and not populate the values?
+        preexistingConditions["hasPreexistingConditions"] = True
+        commorbidities = {
+                        "Diabetes Mellitus": "diabetes mellitus", 
+                        "Cardiopatia": "heart disease",
+                        "Doença de Aparelho Digestivo": "gastrointestinal system disease",
+                        #This was the closest I could find in the DO - 
+                        #somewhat broader than just hematologic disease which is direct translation
+                        "Doença Hematológica": "hematopoietic system disease",
+                        "Doença Hepática": "liver disease",
+                        "Doença Neurológica": "nervous system disease",
+                        "Doença Renal": "kidney disease",
+                        "Doença Respiratória": "respiratory system disease",
+                        "Neoplasia": "neoplasm",
+                        "Etilismo": "alcohol use disorder",
+                        "Obesidade": "obesity",
+                        "Transtorno mental": "mental or behavioural disorder",
+                        "Hipertensão": "hypertension"
+                        }
+        
+        commorbidities_list = []
+
+        for key in commorbidities:
+            if key in raw_commorbidities:
+                commorbidities_list.append(commorbidities[key])
+
+        preexistingConditions["values"] = commorbidities_list
+        return preexistingConditions
+    else:
+        return None         
+
+def convert_location(raw_entry: str):
+    query = ", ".join(word for word in [raw_entry, "Paraíba", "Brazil"] if word)
+    return {"query": query}
+
+def convert_notes(raw_commorbidities: str):
+    raw_notes = []
+    if "Imunossupressão" in raw_commorbidities:
+        raw_notes.append("Patient with immunosupression")
+    if "Tabagismo" in raw_commorbidities:
+        raw_notes.append("Smoker")
+    notes = (', ').join(raw_notes)
+    
+    if notes != '':
```

Comment:
you can just return an empty string for notes, it doesn't matter in the end (same as None).